ocean_lib/models/v4/test/test_factory_router.py

```diff
@@ -0,0 +1,444 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+from re import A
+import hexbytes
+from toolz.itertoolz import cons
+from web3.main import Web3
+from tests.resources.helper_functions import (
+    get_publisher_wallet,
+    get_consumer_wallet,
+    get_another_consumer_wallet,
+)
+
+import pytest
+
+from web3 import exceptions
+from ocean_lib.web3_internal.wallet import Wallet
+
+from ocean_lib.models.v4.erc721_factory import ERC721FactoryContract
+from ocean_lib.models.v4.erc721_token import ERC721Token
+from ocean_lib.models.v4.factory_router import FactoryRouter
+from ocean_lib.web3_internal.contract_utils import get_contracts_addresses
+from ocean_lib.models.v4.bfactory import BFactory
+from ocean_lib.models.v4.erc20_token import ERC20Token
+from ocean_lib.models.v4.models_structures import ErcCreateData
+from ocean_lib.web3_internal.wallet import Wallet
+from tests.resources.helper_functions import get_factory_deployer_wallet
+from ocean_lib.config import Config
```

Comment:
Remove unsed import, you have `config` fixture that you added as a parameter to every function.